template/project.yml

```diff
@@ -40,6 +40,24 @@ drush:
     # The default drush alias to be used when no environment is specified.
   default_alias: ${drush.aliases.local}
 
+# Custom tasks that are triggered at pre-defined times in the build process.
+# Available keys are frontend-build, post-deploy-build, and post-setup-build.
+tasks:
+  # Executed when front end assets should be generated.
+  frontend-build:
+    # E.g., ${docroot}/sites/all/themes/custom/mytheme.
+    dir: ${docroot}
+    # E.g., `npm install` or `bower install`.
+    command: echo 'Hello world!'
```

Comment:
I'm not crazy about "Hello world" here.

I think it could be confusing to see it go past during a build, out of context.

I'd rather see something like `echo 'Frontend build not configured.`